distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +432,63 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Sets the clientAuth configuration in catalog-ftp feature.
+     *
+     * @throws Exception
+     */
+    private void setClientAuthConfiguration(String clientAuth) throws Exception {
+        Configuration config =
+                getAdminConfig().getConfiguration("ddf.catalog.ftp.FtpServerStarter");
+        config.setBundleLocation("mvn:ddf.catalog/ftp/" + System.getProperty("ddf.version"));
+        Dictionary properties = new Hashtable<>();
```

Comment:
You might want to pull the properties from the config and update them instead of making a brand new table so that any previous properties are kept.